spec/mailers/previews/report_mailer_preview.rb

```diff
@@ -1,4 +1,13 @@
 class ReportMailerPreview < ActionMailer::Preview
+  def deleted_user_accounts_report
+    ReportMailer.deleted_user_accounts_report(
+      email: 'test@example.com',
+      name: '',
+      issuers: [],
+      data: [],
+    )
+  end
```

Comment:
This is not a very useful preview, but I don't know enough about how this report email is meant to be generated to give it useful content. Suggestions welcome.

Reply:
`data` is a CSV string 😬 
```suggestion
  def deleted_user_accounts_report
    ReportMailer.deleted_user_accounts_report(
      email: 'test@example.com',
      name: '',
      issuers: ['test-sp-1', 'test-sp-2'],
      data: <<~CSV
        2023-01-01T:00:00:00Z,00000000-0000-0000-0000-000000000000
      CSV
    )
  end
```